ISSUE_TEMPLATE/next.md

```diff
@@ -37,6 +41,8 @@ upgrade` will have the new update.
 - [ ] Post a link to the job as a comment to this issue
 - [ ] Wait for job to finish
 - [ ] Verify that the OSTree commit and its signature are present and valid by booting a VM at the previous release (e.g. `cosa run --qemu-image /path/to/previous.qcow2`) and verifying that `rpm-ostree upgrade --bypass-driver` works and `rpm-ostree status` shows a valid signature.
+    - [ ] x86_64
+    - [ ] aarch64
```

Comment:
This isn't easy to do for folks with no easy access to aarch64? Hmm, maybe let's just leave it off of here for now and I'll remove that item anyway as part of https://github.com/coreos/fedora-coreos-streams/pull/352.

Reply:
I used aws to test aarch64. I think people who are part of release process can have access to aws and can perform testing there.

Reply:
That's a good point. If we want to recommend that, maybe let's add an easy command to copy/paste to spawn the instance so it's easier for folks.

Reply:
I think I'm with jonathan on this one. It would be nice to check, but I think this won't scale past aarch64 because there won't be an easy aws call for ppc64le or s390x. I'll drop it for now and we'll rely on #352 to check it.

Reply:
I'll just add the steps but also mark it as optional. When we get #352 we can remove it.